docs/tutorial.md

```diff
@@ -0,0 +1,103 @@
+
+ After the approve transaction is confirmed I can eventually call `insertCLNtoMarketMaker`. I do the same process again, this time for `CurrencyFactory`, after ABI is loaded I select `insertCLNtoMarketMaker` function. There's actually two functions with the same name (this is Solidity's function overloading feature), and I need the one that received both `token` and `clnAmount`. Maybe we talk latter about the second one. In the token field I paste the `CommunityCurrency`'s address, and put the same 1000 * 1e18 for the ammount field.
+
+Viewing the [transaction](https://ropsten.etherscan.io/tx/0x350fe7bad490baa8a0446c8f5f76bb913b8238fcd882832bb7b4b3e354d1b9c6) you may think some complex stuff happened there. Well, I'll try to sum it up.
+
+- `CurencyFactory` transfers the CLN to itself. He can do it cause I gave him an allowance for this before.
+- Then he transfers the tokens to `MarketMaker` letting him do his mathematical magic. More techically speaking he approves `MM` to use `CLN`, and calls `MM`'s  `change` function.
+- In return the `CurencyFactory` receives `CC` tokens.
+- He sends the tokens back to the sender.
+
+As you see, `CurrencyFactory` It's really just an intermediary that calls `MarketMaker` and holds some data about issuances. The bottom line is I've got around 1,139 `CC` for 1000 `CLN`. Now I'll add my `CC` token to MyEtherWallet, so I can easily see my tokens. I encourage you to do this with your Community Currency also, just for fun as I say.
+
+You can call `insertCLNtoMarketMaker` multiple times, exchanging more CLN for your home-baked Community Currency. By the way did you noticed that the CLN/CC exchange rate changes? This is because as you exchange more CLN for CC, your CC becomes more valuable and so It's price grows. That's how `MarketMaker` works, we'll get into it in the second part of the tutorial.
+
+You call `extractCLNfromMarketMaker` to get your CLN back. If you exchange all you CC to CLN you'll get the innitial CLN ammount, no CLN lost.
+
+Only token issuer can call `insertCLNtoMarketMaker` and `extractCLNtoMarketMaker`. For everyone else to be able to exchange CLN/CC we need to open the CC's `MarketMaker` for public usage. This is done to give sufficient time for the currency issuer. As I explained before CLN/CC rate depends on the CC demand. When the token just created the demand for CC is low, so the currency issuer has an advantage to buy CLN for a cheapest price. Other preparations might not be related to crypto at all.
```

Comment:
* When the token just created --> When the token is just created
* for a cheapest --> for the cheapest